be/src/storage/rowset/column_writer.cpp

```diff
@@ -573,6 +573,18 @@ Status ScalarColumnWriter::finish_current_page() {
             PageIO::compress_page_body(_compress_codec, _opts.compression_min_space_saving, body, &compressed_body));
     if (compressed_body.size() == 0) {
         // page body is uncompressed
+        double space_saving =
+                1.0 - static_cast<double>(encoded_values->size()) / static_cast<double>(encoded_values->capacity());
```

Comment:
It's better to add a concise message about the optimization result.
The result can make a clear indication about to the users.